packages/babel-plugin-transform-async-to-promises/src/index.js

```diff
@@ -0,0 +1,113 @@
+/* Babel 7 transform */
+
+const babylon = require("babylon");
+const generate = require("@babel/generator").default;
```

Comment:
This dep doesn’t seem to be listed in package.json

Reply:
Are those deps not managed by lerna; or do you still need to explicitly list them?

(I never looked up how lerna actually manages these...)

Reply:
I'm sure lerna might fix it so it *happens* to work, but you should always explicitly list every dep.

In this case, if it's not explicitly listed, *consumers* will break (since this isn't test code).

Reply:
Finally got around to updating this is https://github.com/babel/babel/pull/7076/commits/943a44c522dca0acaabf3d1b73317ab155124f99. Hopefully this resolve the remaining issues for this implementation.